requirements.in

```diff
@@ -40,3 +40,5 @@ python-dateutil>=2.8.1
 jsonschema>=3.2.0
 importlib-metadata==1.7.0
 django-celery-results
+torch==2.0.1
+transformers==4.29.2
```

Comment:
Could you remove these from the production requirements? When we deploy it, we can re-add them. You already added them to the dev requirements